torch/_dynamo/eval_frame.py

```diff
@@ -530,6 +531,10 @@ def export(
         assert (
             aten_graph
         ), "Specifying a decomposition_table table or tracing mode is illegal without setting aten_graph=True"
+
+    if aten_graph and decomposition_table is None:
+        decomposition_table = core_aten_decompositions
```

Comment:
one problem with setting up the api this way is that the user needs to know that if they want to just make a slight tweak to the default decomps, they have to know to clone core aten decomps first and then twiddle it

Reply:
How about `get_core_aten_decompositions()` to always return a cloned decomp_table?

Reply:
Well, I suppose that is some helpful QOL (but I wouldn't call it `get_` anymore), but the documentation point (at least) still stands